packages/base/lib/plugin.js

```diff
@@ -41,11 +43,13 @@ export default class BasePlugin {
    * In order to add a new route to Appium with this plugin. Or, you could add new listeners to the
    * httpServer object.
    *
-   * @param {object} expressApp - the Express 'app' object used by Appium for route handling
-   * @param {http.Server} httpServer - the node HTTP server that hosts the app
+   * @param {import('express').Application} expressApp - the Express 'app' object used by Appium for route handling
+   * @param {import('http').Server} httpServer - the node HTTP server that hosts the app
    */
-  /*static async updateServer (expressApp, httpServer) {
-  }*/
+  // eslint-disable-next-line no-unused-vars, require-await
+  static async updateServer (expressApp, httpServer) {
```

Comment:
this will break existing plugins which have chosen not to implement this function. if it exists, it will be called. and so uncommenting this means there is a requirement for plugin authors to update their plugins to implement it as a no-op.

if we need to uncomment it i guess we should make it a no-op instead of an error, though this will cause us to lose information in the appium logs (which currently can decide whether a given plugin is actually going to update the server or not).

Reply:
yeah. I already made this a no-op.